### ISSUE TITLE
S2325 is not being detected

### ISSUE THREAD

**Issue by alexeyzimarev** [SonarSource/sonar-csharp#1285]
When running the Sonar C# using the Sonar MSBuild scanner, the S2325, for which I deliverately put the code in my PR, is not being detected.

The code is:

```csharp
    public class blahBlah
    {
        public string Prop;
        
        public string What_ever()
        {
            return "blkah";
        }
    }
```

The rule is enabled in my custom quality profile on SonarCloud.io. Logs shows that the custom profile is being used.
![selection_058](https://user-images.githubusercontent.com/2821205/38028286-d162d0ec-3292-11e8-9ac6-cf4d30502ed4.png)

```

28-Mar-2018 13:41:20 | INFO: -------------  Scan project
-- | --
28-Mar-2018 13:41:20 | INFO: Load server rules
28-Mar-2018 13:41:20 | INFO: Load server rules (done) \| time=266ms
28-Mar-2018 13:41:20 | INFO: Base dir: D:\bamboo-home\xml-data\build-dir\EV-PROC15-JOB1\src\project
28-Mar-2018 13:41:20 | INFO: Working dir: D:\bamboo-home\xml-data\build-dir\EV-PROC15-JOB1\.sonarqube\out\.sonar\project_770BF04F-3E6F-4F66-AFE0-CE1E0B2DFA51
28-Mar-2018 13:41:20 | INFO: Source paths: blahBlah.cs, CacheCalls.cs, DecodeUtils.cs, Properties/AssemblyInfo.cs, UsageLogDecodeException.cs, DecodeException.cs, TsqlLegacy.cs, paket.references
28-Mar-2018 13:41:20 | INFO: Source encoding: UTF-8, default locale: en_GB
28-Mar-2018 13:41:21 | INFO: Index files
28-Mar-2018 13:41:21 | INFO: 8 files indexed
28-Mar-2018 13:41:21 | INFO: Quality profile for cs: ABAX way
28-Mar-2018 13:41:24 | INFO: Sensor C# Properties [csharp]
28-Mar-2018 13:41:24 | INFO: Sensor C# Properties [csharp] (done) \| time=0ms
28-Mar-2018 13:41:24 | INFO: Sensor SonarJavaXmlFileSensor [java]
28-Mar-2018 13:41:24 | INFO: Sensor SonarJavaXmlFileSensor [java] (done) \| time=0ms
28-Mar-2018 13:41:24 | INFO: SCM provider for this project is: git
28-Mar-2018 13:41:24 | INFO: 2 files to be analyzed
28-Mar-2018 13:41:25 | INFO: 2/2 files analyzed
```

**Comment by Evangelink** [comment]
Hi @alexeyzimarev,

Sorry for this late answer. Could you let me know the version of your scanner for msbuild? I can't reproduce the same issue here.

**Comment by alexeyzimarev** [comment]
I need to try again now. I was using the latest scanner version available for the date of the issue.

**Comment by Evangelink** [comment]
Hi @alexeyzimarev,

Sorry for the ping but have you managed to give it a try? Are you still experiencing issue?

**Comment by Evangelink** [comment]
Hi @alexeyzimarev,

Sorry to ping again but I'd like to know if you've been able to give it a new try?

**Comment by Evangelink** [comment]
Hi @alexeyzimarev,

I am closing this ticket. Please feel free to re-open or create a new issue if you encounter the issue again.
